lib/crawly/manager.ex

```diff
@@ -95,45 +115,15 @@ defmodule Crawly.Manager do
        tref: tref,
        prev_scraped_cnt: 0,
        workers: worker_pids
-     }, {:continue, {:startup, options}}}
+     }, {:continue, {:startup, {spider_name, async_start_req}}}}
```

Comment:
Passing args to the handle_continue `:startup` in this manner limits future extensions. Seems like an unnecessary change.